components/family-studies/ui/src/main/resources/PhenoTips/FamilyContentTopMenu.xml

```diff
@@ -358,19 +358,9 @@
   var exportLinks = $$(' .export-link');
   exportLinks.invoke('observe', 'click', function (event) {
     event.stop();
-    if (typeof event.findElement('.editbody') !== "undefined") {
-      // We are in edit mode.
-      var myWindow = window.open("","_blank");
-      var exportAfterSave = function() {
-        myWindow.location = event.findElement().href;
-        document.stopObserving('xwiki:document:saved', exportAfterSave);
-      };
-      document.observe('xwiki:document:saved', exportAfterSave);
-      document.fire('xwiki:actions:save', {'continue' : true, 'form' : event.findElement('form')});
-    } else {
-      // View mode
+    PhenoTips.widgets.FormUtils.getFormState().saveIfFormDirty(function () {
       window.open(event.findElement().href,"_blank");
```

Comment:
Possible solution:

    event.stop();
    var myWindow = window.open("", "_blank");
    PhenoTips.widgets.FormUtils.getFormState().saveIfFormDirty(function () {
      myWindow.location = event.findElement().href;
    });

The same must be done in `ContentTopMenu` too.